public/video-ui/src/pages/Video/index.jsx

```diff
@@ -277,11 +277,17 @@ class VideoDisplay extends React.Component {
             this.getVideo();
           }}
           onSave={() => {
-            this.updateEditingState({
-              key: 'editingFurniture',
-              editing: false
-            });
-            this.saveAndUpdateVideo(video);
+            this.saveAndUpdateVideo(video)
+              .then(() => {
+                this.updateEditingState({
+                  key: 'editingFurniture',
+                  editing: false
+                });
+              })
+              .catch(() => {
+                // Keep editing state as true on save failure
```

Comment:
Probably out of scope for this PR, but no we have access to the error, we could possibly improve the user feedback by triggering some transient messaging specific to the last click to compliment the main error bar - IE telling them what happened for the last click of the save button. 
This might be most useful for missing/invalid fields where the user might not have noticed what was wrong.

I'm thinking of a pattern something like https://mui.com/material-ui/react-snackbar/#supplementary-components